lib/backend/k8s/syncer.go

```diff
@@ -285,7 +293,12 @@ func (syn *kubeSyncer) performSnapshot() ([]model.KVPair, map[model.Key]bool, re
 
 		// Get NetworkPolicies (Policies)
 		log.Info("Syncing NetworkPolicy")
-		npList, err := syn.kc.clientSet.NetworkPolicies("").List(opts)
+		npList := extensions.NetworkPolicyList{}
+		err = syn.kc.clientSet.Extensions().RESTClient().
+			Get().
+			Resource("networkpolicies").
+			Timeout(10 * time.Second).
```

Comment:
Might be nice to have these times declared as costs above rather than buried here.